theme/static/css/hydroshare_core.css

```diff
@@ -3077,7 +3079,7 @@ h4.title {
 }
 
 .carousel-inner {
-	background: #6D6D6D;
+	/*background: #6D6D6D;*/
 }
 
```

Comment:
@mauriel133 Now that the only ```background``` style definition is commented out, should this ```carousel-inner``` class be deleted?